src/hooks/useActiveDescendantListFocus/useActiveDescendantListFocusHooks.js

```diff
@@ -0,0 +1,242 @@
+import { useCallback, useEffect, useMemo, useState } from "react";
+import useKeyEvent from "../useKeyEvent";
+import useEventListener from "../useEventListener";
+import usePrevious from "../usePrevious";
+import { getNextSelectableIndex, getPreviousSelectableIndex } from "./useActiveDescendantListFocusHelpers";
+import { useListenFocusTriggers } from "hooks/useListenFocusTriggers";
+
+const ARROW_DIRECTIONS = {
+  UP: "ArrowUp",
+  DOWN: "ArrowDown",
+  RIGHT: "ArrowRight",
+  LEFT: "ArrowLeft"
+};
+
+const ENTER_KEY = "Enter";
+const SPACE_KEY = " ";
+
+export function useSupportArrowsKeyboardNavigation({
```

Comment:
same code, only moved getNextSelectableIndex, getPreviousSelectableIndex to helpers